lib/ckeditor/plugins/dialog/styles/dialog.css

```diff
@@ -10,7 +10,7 @@
 	height: 100%;
 	top: 0;
 	left: 0;
-	z-index: 10010;
+	z-index: 999999 !important;

```

Comment:
Could you use option [baseFloatZIndex](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-baseFloatZIndex) in config instead?

Reply:
Done, thanks!